src/ansys/dpf/core/scoping.py

```diff
@@ -251,73 +278,79 @@ def _get_index(self, scopingid):
         """
         return self._api.scoping_index_by_id(self, scopingid)
 
-    def id(self, index: int):
-        """Retrieve the ID at a given index.
+    def id(self, index: int) -> int:
+        """Retrieve the entity ID at a given index.
 
         Parameters
         ----------
-        index : int
-            Index for the ID.
+        index:
+            Index of the entity in the scoping.
 
         Returns
         -------
-        size : int
+        entity_id:
+            ID of the entity at index in the scoping.
 
         """
         return self._get_id(index)
 
-    def index(self, id: int):
-        """Retrieve the index of a given ID.
+    def index(self, id: int) -> int:
+        """Retrieve the index for a given entity ID.
 
         Parameters
         ----------
-        id : int
-            ID for the index to retrieve.
+        id:
+            Entity ID at the index in the scoping.
 
         Returns
         -------
-        size : int
+        index:
+            Index in the scoping for the entity ID.
 
         """
         return self._get_index(id)
 
     @property
-    def ids(self):
+    def ids(self) -> Union[DPFArray, list[int]]:
         """Retrieve a list of IDs in the scoping.
 
         Returns
         -------
         ids : DPFArray, list of int
-            List of IDs to retrieve. By default a mutable DPFArray is returned, to change
-            the return type to a list for the complete python session, see
+            List of IDs to retrieve. By default, a mutable DPFArray is returned.
+            To change the return type to a list for the complete python session, see
             :func:`ansys.dpf.core.settings.get_runtime_client_config` and
             :func:`ansys.dpf.core.runtime_config.RuntimeClientConfig.return_arrays`.
             To change the return type to a list once, use
             :func:`ansys.dpf.core.scoping.Scoping._get_ids` with the parameter ``np_array=False``.
 
         Notes
         -----
-        Print a progress bar.
+        Prints a progress bar.  # TODO is that true?
         """
         return self._get_ids()
 
     @ids.setter
-    def ids(self, value):
+    def ids(self, value: IdVectorType):
         self._set_ids(value)
 
     @property
-    def location(self):
-        """Location of the IDs as a string, such as ``"nodal"``, ``"elemental"``, and ``"time_freq"``.
+    def location(self) -> str:
+        """Location of the scoping as a string.
+
+        This defines the type of entity the IDs stored correspond to.
```

Comment:
I am having a bit of trouble parsing this. Is it defining the type of entity stored in an ID? Defining an entity corresponding to a stored ID?

Reply:
@JennaPaikowsky this defines what type the entities are. The entity IDs can be node IDs, element IDs, face IDs, etc...
Not sure how to formulate that properly.

Reply:
Ah, got it.

"This defines what type of entity the IDs correspond to (such as node ID, element ID, face ID, and so on)."

Reply:
```suggestion
        This defines the type of entity the IDs correspond to (such as node ID, element ID, face ID, and so on).
```